lib/srv/desktop/deskproto/proto.go

```diff
@@ -307,6 +310,43 @@ func decodeClientUsername(in peekReader) (ClientUsername, error) {
 	return ClientUsername{Username: username}, nil
 }
 
+// MouseWheelAxis identifies a scroll axis on the mouse wheel.
+type MouseWheelAxis byte
+
+const (
+	VerticalWheelAxis   = MouseWheelAxis(0)
+	HorizontalWheelAxis = MouseWheelAxis(1)
+)
+
+// MouseWheel is the mouse wheel scroll message.
+// https://github.com/gravitational/teleport/blob/master/rfd/0037-desktop-access-protocol.md#8---mouse-wheel
+type MouseWheel struct {
+	Axis  MouseWheelAxis
+	Delta int32
```

Comment:
Not a huge deal, but I would consider putting delta first to ensure aligned reads and potentially save on padding if we expand this message in the future.